src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/EncoderParameters.cs

```diff
@@ -84,12 +84,12 @@ internal static EncoderParameters ConvertFromMemory(IntPtr memory)
             int count = Marshal.ReadInt32(memory);
 
             EncoderParameters p = new EncoderParameters(count);
-            int size = Marshal.SizeOf(typeof(EncoderParameter));
+            int size = Marshal.SizeOf<EncoderParameter>();
             long arrayOffset = (long)memory + IntPtr.Size;
 
             for (int i = 0; i < count; i++)
             {
-                Guid guid = (Guid)Marshal.PtrToStructure((IntPtr)(i * size + arrayOffset), typeof(Guid))!;
+                Guid guid = Marshal.PtrToStructure<Guid>((IntPtr)(i * size + arrayOffset))!;
```

Comment:
Most (if not all) paths touched in this PR are using types that are blittable or that can be trivially made blittable. It would be nice to avoid `Marshal.*` all together and just use pointers.

For example, this line can be just `Guid guid = *(Guid *)(i * size + arrayOffset);`